cores/arduino/USBCore.cpp

```diff
@@ -913,4 +955,9 @@ void* epBuffer(unsigned int n)
 {
     return (void*)EPBuffers().desc(n);
 }
+
+bool USBCore_::isSuspended()
+{
+    return USBCore().usbDev().cur_status == USBD_SUSPENDED;
+}
```

Comment:
this method doesn't seem to be being used

Reply:
This was part of an already-integrated commit to support the HostPowerManagement plugin in Kaleidoscope.